index.js

```diff
@@ -154,14 +162,19 @@ const TinderCard = React.forwardRef(({ flickOnSwipe = true, children, onSwipe, o
     }
   }))
 
-  const handleSwipeReleased = React.useCallback(async (element, speed) => {
+  const handleSwipeReleased = React.useCallback(async (element, speed, lastLocation) => {
     if (swipeAlreadyReleased.current) { return }
     swipeAlreadyReleased.current = true
 
     // Check if this is a swipe
-    if (Math.abs(speed.x) > settings.swipeThreshold || Math.abs(speed.y) > settings.swipeThreshold) {
-      const dir = getSwipeDirection(speed)
+    let checkMore = false
```

Comment:
```suggestion
    let swipeDistanceThresholdMet = false
```

a little more readable